R/pkg/R/DataFrame.R

```diff
@@ -194,7 +194,12 @@ setMethod("isLocal",
 setMethod("showDF",
           signature(x = "SparkDataFrame"),
           function(x, numRows = 20, truncate = TRUE) {
-            s <- callJMethod(x@sdf, "showString", numToInt(numRows), truncate)
+            truncate2 <- as.numeric(truncate)
+            if (truncate2 == 1) {
```

Comment:
Related to the other comment you can use `is.logical(truncate)` to see if `truncate` is a boolean in R